src/main/java/uk/gov/hmcts/reform/ccd/client/CoreCaseDataApi.java

```diff
@@ -25,31 +25,35 @@
         configuration = CoreCaseDataConfiguration.class)
 public interface CoreCaseDataApi {
 
+    String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```

Comment:
consider reusing this in the `CaseAccessApi` please

Reply:
done